CMakeLists.txt

```diff
@@ -1,479 +1,44 @@
-# cmake global
 cmake_minimum_required(VERSION 3.0.0)
 
-# Figure out environment.
-set(ETH_CMAKE_DIR   "${CMAKE_CURRENT_LIST_DIR}/../cpp-ethereum-cmake"   CACHE PATH "The path to the cmake directory")
+set(ETH_CMAKE_DIR "${CMAKE_CURRENT_LIST_DIR}/cmake" CACHE PATH "The path to the cmake directory")
```

Comment:
I think `"${CMAKE_SOURCE_DIR}/cmake"` would be much clearer to express the submodules are direct subdirectories.